docs/content/tutorials/helm/index.md

```diff
@@ -0,0 +1,654 @@
+
+In this step you will:
+
+- Add Redis to the application using a Recipe.
+- Update the Kubernetes secret with the connection information from Redis.
+
+First, check recipes installed in your environment by running:
+
+```
+rad recipe list
+```
+
+You will see output like this:
+
+```
+NAME      TYPE                                    TEMPLATE KIND  TEMPLATE VERSION  TEMPLATE
+default   Applications.Datastores/sqlDatabases    bicep                            radius.azurecr.io/recipes/local-dev/sqldatabases:latest
+default   Applications.Messaging/rabbitMQQueues   bicep                            radius.azurecr.io/recipes/local-dev/rabbitmqqueues:latest
+default   Applications.Dapr/pubSubBrokers         bicep                            radius.azurecr.io/recipes/local-dev/pubsubbrokers:latest
+default   Applications.Dapr/secretStores          bicep                            radius.azurecr.io/recipes/local-dev/secretstores:latest
+default   Applications.Dapr/stateStores           bicep                            radius.azurecr.io/recipes/local-dev/statestores:latest
+default   Applications.Datastores/mongoDatabases  bicep                            radius.azurecr.io/recipes/local-dev/mongodatabases:latest
+default   Applications.Datastores/redisCaches     bicep                            radius.azurecr.io/recipes/local-dev/rediscaches:latest
+```
+
+The recipe for `Applications.Datastores/redisCaches` is the one that will be used in this example.
+
+{{< alert title="💡 Recipes" color="info" >}}
+ Radius includes Recipes for local development when you use `rad init`. These **local-dev** Recipes run popular OSS technologies as containerized infrastructure without requiring a cloud account.
+ 
+ In a production environment you can substitute recipes that will create cloud or on-premises dependencies instead.
+{{< /alert >}}
+
+Make sure the `app.yaml` file from `./demo/Chart/templates/app.yaml` is open in your editor. At the bottom of the file add the following text, including the `---`:
+
+```yaml
+---
+apiVersion: radapp.io/v1alpha3
+kind: Recipe
+metadata:
+  name: db
+  namespace: {{ .Release.Namespace }}
+spec:
+  type: Applications.Datastores/redisCaches
+  secretName: redis-secret
+```
+
+Defining a `Recipe` object in Kubernetes will use a Radius Recipe to create dependencies for your application:
+
+- The `.spec.type` field defines the type of resource to create. `Applications.Datastores/redisCaches` is the type for a Redis Cache.
+- The `.spec.secretName` field tells Radius where to store connection information. This is optional, and should be used to interoperate with other Kubernetes technologies that read from secrets. In our case we're using the secret to populate an environment variable.
+
+Now that you have made the edits, save the file, and deploy the application again using Helm. Since the namespace and secret have already been created, we only need to run the `helm` command.
```

Comment:
```suggestion
Save the file after you have made the edits and deploy the application again using Helm. Since the namespace and secret have already been created, you only need to run the `helm` command.
```